nette/notejam/app/forms/Note/EditNoteFormFactory.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Forms\Note;
+
+use App\Model\NoteManager;
+use App\Model\PadManager;
+use Nette;
+use Nette\Application\UI\Form;
+use Nette\Utils\ArrayHash;
+
+
+class EditNoteFormFactory extends Nette\Object
+{
+
+	/** @var PadManager */
+	private $padManager;
+
+	/** @var NoteManager */
+	private $noteManager;
+
+	/**
+	 * EditPadFormFactory constructor.
+	 * @param PadManager  $padManager
+	 * @param NoteManager $noteManager
+	 */
+	public function __construct(PadManager $padManager, NoteManager $noteManager)
+	{
+		$this->padManager = $padManager;
+		$this->noteManager = $noteManager;
+	}
+
+	/**
+	 * @param int    $id
+	 * @param string $name
+	 * @param string $text
+	 * @param int    $pad
+	 * @return Form
+	 */
+	public function create($id, $name, $text, $pad)
```

Comment:
passing the fields through arguments doesn't scale very well, even through for few fields it might look good